fs.go

```diff
@@ -74,6 +76,18 @@ func isDirExist(dirname string) (bool, error) {
 	return fi.IsDir(), nil
 }
 
+func (s fsStorage) isBucketExists(bucketName string) (isExist bool, err error) {
```

Comment:
storage shouldn't have a concept of buckets, it should be named  - `isVolumeExists(volume string)`